File: SoloLearn/Carrot Cake/CarrotCake.swift
if let carrots = readLine(), let boxes = readLine(), let c = Int(carrots), let b = Int(boxes) {
    let remainder = c % b
    let missing = 7 - remainder
    if missing <= 0 {
        print("Cake Time")
    } else {
        print("I need to buy \(missing) more")
    }
}

File: SoloLearn/Carrot Cake/CarrotCake.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace SoloLearn
{
    class Program
    {
        static void Main(string[] args)
        {
            int carrots = Convert.ToInt32(Console.ReadLine()), boxes = Convert.ToInt32(Console.ReadLine()), remainder = carrots % boxes, missing = 7 - remainder;
            if (missing <= 0)
                Console.WriteLine("Cake Time");
            else    
                Console.WriteLine($"I need to buy {missing} more");
        }
    }
}